front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -841,6 +859,34 @@ export default function AppChat({
     while (m.pop()?.role !== "user" && m.length > 0);
   }
 
+  function logFeedback(message: ChatMessageType): void {
+    const chatSessionUrl = `https://dust.tt/w/${owner.sId}/u/chat/${chatSession.sId}`;
+
+    // gets the user message preceding `message` in the chat session
+    const messagesCopy = [...messages];
+    let previousUserMessage: ChatMessageType | undefined;
+    while (messagesCopy.pop() !== message);
+    while ((previousUserMessage = messagesCopy.pop())?.role !== "user");
+
+    // create the logging message
+    const loggingMessage = {
```

Comment:
We don't need to log the message. If we need to introspect messages we'll build an UI for us internally that will  let us do that introspection since we have the messages in DB. Datadog really is not great for that and not meant to be a store of data (we have a retention policy there anyway).